unison-cli/src/Unison/Codebase/Editor/Slurp.hs

```diff
@@ -227,64 +201,36 @@ computeVarStatuses depMap varReferences codebaseNames =
        in case ld of
             LD.TypeReference _typeRef ->
               case Set.toList existingTypesAtName of
-                [] -> DefOk New
+                [] -> New
                 [r]
-                  | LD.typeRef r == ld -> DefOk Duplicated
-                  | otherwise -> DefOk Updated
-                -- If there are many existing types, they must be in conflict.
-                -- Currently we treat conflicts as errors rather than resolving them.
-                _ -> DefErr Conflict
+                  | LD.typeRef r == ld -> Duplicated
+                  | otherwise -> Updated
+                _ -> Updated
             LD.TermReference {} ->
               case Set.toList existingTermsOrCtorsAtName of
-                [] -> DefOk New
-                rs | any Referent.isConstructor rs -> DefErr TermCtorCollision
+                [] -> New
+                rs | any Referent.isConstructor rs -> TermCtorCollision
                 [r]
-                  | LD.referent r == ld -> DefOk Duplicated
-                  | otherwise -> DefOk Updated
-                -- If there are many existing terms, they must be in conflict.
-                -- Currently we treat conflicts as errors rather than resolving them.
-                _ -> DefErr Conflict
+                  | LD.referent r == ld -> Duplicated
+                  | otherwise -> Updated
+                _ -> Updated
```

Comment:
I notice here there's a possible case where we have a conflict, but also one of the new definitions is a duplicate of something;

I think the behaviour you have here makes sense, just roll it into an Update for simplicity sake; but just wanted to point it out in case we wanted to do something differently.